common/beerocks/bwl/CMakeLists.txt

```diff
@@ -83,13 +98,50 @@ elseif(BWL_TYPE STREQUAL "WAV")
 
 elseif(BWL_TYPE STREQUAL "DWPAL")
     
+    include_directories(
+        ${HOSTAPD_DIR}/src/drivers
+        ${PLATFORM_BUILD_DIR}/iwlwav-iw-4.14
+        ${PLATFORM_INCLUDE_DIR}/libnl3
+    )
```

Comment:
I don't see how this change is related to "add cmake turris-omnia support" (in other words, it should have been a separate commit).

(nitpick, not really needed to change this.)